linux_os/guide/system/accounts/accounts-pam/enable_pam_namespace/rule.yml

```diff
@@ -18,6 +18,8 @@ rationale: |-
 
 severity: low
 
+platform: not osbuild
```

Comment:
This rule already has a `platform` key, please have only one `platform` key in a rule.

Reply:
Right.